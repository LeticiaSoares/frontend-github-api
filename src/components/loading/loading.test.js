import React from 'react'
import { shallow } from 'enzyme';
import Loading from './index'
import 'jest-styled-components'

describe('<Loading />', () => {
    it('should match default snapshot', () => {
        const component = shallow(<Loading />)
        expect(component).toMatchSnapshot()
    })
    it('should have the right displayName', () => {
        expect(Loading.displayName).toEqual('Loading')
    })
})
