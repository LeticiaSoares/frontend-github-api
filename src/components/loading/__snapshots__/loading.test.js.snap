// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> should match default snapshot 1`] = `ShallowWrapper {}`;
